1080i/Includes.xml

```diff
@@ -4586,15 +4604,124 @@
+        <onfocus condition="Integer.IsGreater(Container(301125).NumItems,0) + Control.IsVisible(301125)">SetFocus(301125,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301126).NumItems,0) + Control.IsVisible(301126)">SetFocus(301126,0,absolute)</onfocus>
+
+        <onfocus condition="Integer.IsGreater(Container(301131).NumItems,0) + Control.IsVisible(301131)">SetFocus(301131,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301132).NumItems,0) + Control.IsVisible(301132)">SetFocus(301132,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301133).NumItems,0) + Control.IsVisible(301133)">SetFocus(301133,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301134).NumItems,0) + Control.IsVisible(301134)">SetFocus(301134,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301135).NumItems,0) + Control.IsVisible(301135)">SetFocus(301135,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301136).NumItems,0) + Control.IsVisible(301136)">SetFocus(301136,0,absolute)</onfocus>
+
+        <onfocus condition="Integer.IsGreater(Container(301141).NumItems,0) + Control.IsVisible(301141)">SetFocus(301141,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301142).NumItems,0) + Control.IsVisible(301142)">SetFocus(301142,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301143).NumItems,0) + Control.IsVisible(301143)">SetFocus(301143,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301144).NumItems,0) + Control.IsVisible(301144)">SetFocus(301144,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301145).NumItems,0) + Control.IsVisible(301145)">SetFocus(301145,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301146).NumItems,0) + Control.IsVisible(301146)">SetFocus(301146,0,absolute)</onfocus>
+
+        <onfocus condition="Integer.IsGreater(Container(301151).NumItems,0) + Control.IsVisible(301151)">SetFocus(301151,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301152).NumItems,0) + Control.IsVisible(301152)">SetFocus(301152,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301153).NumItems,0) + Control.IsVisible(301153)">SetFocus(301153,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301154).NumItems,0) + Control.IsVisible(301154)">SetFocus(301154,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301155).NumItems,0) + Control.IsVisible(301155)">SetFocus(301155,0,absolute)</onfocus>
+        <onfocus condition="Integer.IsGreater(Container(301156).NumItems,0) + Control.IsVisible(301156)">SetFocus(301156,0,absolute)</onfocus>
+    </include>
```

Comment:
with the 2 versions there is this part of the code that I am not happy with. 
I looked for a more elegant code but I did not find anything other than this tank to crush a fly. :/